.github/workflows/fake-bpy-ci.yml

```diff
@@ -0,0 +1,84 @@
+name: fake-bpy-module CI
+
+on:
+  push:
+    branches:
+      # Temporarily trigger only on "ci_testing" branch
+      - ci_testing
+  # # Trigger on all PRs. By default, only run if activity type is opened, synchronize, or reopened
+  # pull_request:
+
+jobs:
+  build_modules:
+    name: Build fake-bpy modules
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+
+      - name: Blender downloads cache
+        uses: actions/cache@v2
+        id: blender-bin-cache
+        with:
+          path: blender-bin
+          key: blender-binaries-${{ runner.os }}-${{ hashFiles('**/tools/utils/download_blender.sh') }}
+
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install wheel sphinx
```

Comment:
It would be nicer these packages are moved to `requirements.txt`.

Also, is `sphinx` package still needed?
Recently, I used .rst files to generate fake-bpy-module instead of .xml files.
c.f. https://github.com/nutti/fake-bpy-module/commit/3dac9653d3327b201e532f55d41423b325542685

Reply:
> Also, is sphinx package still needed?

without the `sphinx` package, the pipeline failed to build the last time I tried. I will check again

> It would be nicer these packages are moved to requirements.txt.

Yes, of course. But I did not want to change this file, I expected you left it out of there for  reason. Do you want me to add it?

Reply:
Yes, please add to `requirements.txt` after checking `sphinx` package.

Reply:
While checking for `sphinx`, I opened a PR for `wheel`: https://github.com/nutti/fake-bpy-module/pull/55